### ISSUE TITLE
Larynx needs AVX compatible CPU on amd64. 

### ISSUE THREAD

**Issue by nordeep** [rhasspy/rhasspy#150]
Seems larynx compiled with AVX on amd64. It would be nice to add CPU requirement to documentation!

As example - if you using KVM based VM with kvm64 CPU, you will be surprised - why larynx doesn't work?! :laughing:

**Comment by synesthesiam** [comment]
Yeah, the speed difference without AVX is pretty large. I'm still thinking about how I'm going to do this properly with Larynx.

For now, I've created "no AVX" variants of [my MozillaTTS docker images](https://github.com/synesthesiam/docker-mozillatts). I'll update the Larynx ones when I get a chance.

**Comment by h4de5** [comment]
(!) just ran into the same problem. larynx works on a rpi4 satellite - but not on my home server. looking forward to that "no avx" variant.

**Comment by h4de5** [comment]
@synesthesiam would it be possible to show a message/infobox about this missing AVX cpu feature? something like `this will not work on your CPU` or if you get time to create no AVX variants something like: `this will be slow on your CPU`.

correct me if I am wrong, but I assume the following options are not working on such CPUs:
- microft precise
- mozilla deepspeech
- rasa NLU
- larynx 
- (normal) mozillaTTS

**Comment by synesthesiam** [comment]
For future me: https://detect-simd.readthedocs.io/en/latest/

**Comment by almostserious** [comment]
I believe the same problem is also seen on Deepspeech on AMD64?
Seems like this also requires AVX. Is there also a no AVX branch available?

**Comment by synesthesiam** [comment]
OK, in 2.5.10 I'll put a warning on the settings page for Larynx, DeepSpeech, and Precise. For Rasa and MozillaTTS, I'll hold off since those are external services that could be running anywhere.
